src/Uno.UI.RemoteControl.VS/DebuggerHelper/ProfilesObserver.cs

```diff
@@ -99,7 +98,12 @@ private void TryUpdateSolution()
 	{
 		if (_dte.Solution.SolutionBuild.StartupProjects is object[] newStartupProjects)
 		{
-			if (!newStartupProjects.SequenceEqual(_existingStartupProjects))
+			if (_existingStartupProjects is { Length: 0 })
```

Comment:
nit: `_existingStartupProjects` is never null. This means that a simple length validation should be sufficient.
```suggestion
			if (_existingStartupProjects.Length == 0)
```